### DISCUSSION TITLE
Allow labels created by ConDes to be exported in o65 binary.

### DISCUSSION THREAD

**Discussion by ghost** [cc65/cc65#2048] [Q&A]
My o65 loader needs to relocate the interruptors table to a different address hard-coded in ROM. That would be somewhat easy if o65 supported additional segments, but alas, it doesn't. With the table address at hand, I think the loader could do it, provided that the reloc info of the addresses inside the table are also available.

I'd appreciate any pointers on a way to achieve that with current ld65, even if hacky.

Here's a repro of the failure I get in trying to export a symbol created by ConDes:
[repro.tar.gz](https://github.com/cc65/cc65/files/11125804/repro.tar.gz)

```
$ make
ca65 -o test.o test.s
ld65 -C firmware.cfg -o test.o65 test.o
ld65: Error: Unresolved export: '__IRQ_TABLE__'
make: *** [Makefile:7: test.o65] Error 1
```

**Comment by mrdudz** [comment]
Interesting question.... i don't have a good answer though :)

**Reply by ghost** [reply]
isn't it something that ld65 could do? I mean, it is defining those symbols, I was expecting that they could be exported to o65 binary.
If there's a consensus that this feature could be added, I guess I could give it a try and implement it (in my own pace, though).
